docs/apps/publishing.md

```
@@ -4,65 +4,49 @@ title: "Publishing your app"
 
 There are two different ways of publishing apps (formerly known as add-ons). One is to publish pre-built containers to a container registry and the other option is to have users build the containers locally on their Home Assistant instance.
 
-#### Pre-built containers
+### Pre-built containers
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)